scout/server/blueprints/variant/templates/variant/utils.html

```diff
@@ -271,7 +271,7 @@
           <input type="hidden" name="sample" value="{{case.sample_names|join(',')}}">
           <input type="hidden" name="build" value="{{case.genome_build}}">
 
-          <input type="hidden" name="contig" value="{% if zoom == "start" %}{{ variant.chromosome }}{% elif zoom == "end" %}{{ variant.end_chrom }}{% endif %}">
+          <input type="hidden" name="contig" value="{% if zoom == "start" or zoom == "length" %}{{ variant.chromosome }}{% elif zoom == "end" %}{{ variant.end_chrom }}{% endif %}">
```

Comment:
you could also have if zoom in ["start", "length"], but it works!

Reply:
That is cleaner - I will change!